expo/tests/scheduler/test_view_list.py

```diff
@@ -84,24 +87,27 @@ def test_view_list_default_view_current_conf_exists():
     nt.assert_false(previous_class.title in response.content)
 
 
+@pytest.mark.django_db
 def test_view_list_event_type_not_case_sensitive():
     param = 'class'
-    request1 = RequestFactory().get(
-        reverse("event_list",
-                urlconf="scheduler.urls",
-                args=[param]))
-    request2 = RequestFactory().get(
-        reverse("event_list",
-                urlconf="scheduler.urls",
-                args=[param.upper()]))
+    client = Client()
+    password = "password"
+    url_lower = reverse("event_list",
+                        urlconf="scheduler.urls",
+                        args=[param.lower()])
+
+    url_upper = reverse("event_list",
+                        urlconf="scheduler.urls",
+                        args=[param.upper()])
+    user = ProfileFactory.create().user_object
+    client.login(username=user.username,
+                 email=user.email,
+                 password=password)
```

Comment:
Actually - why login at all here?  This is a public function with no login required.